homeassistant/components/barry/strings.json

```diff
@@ -0,0 +1,24 @@
+{
+  "config": {
+    "error": {
+      "invalid_access_token": "[%key:common::config_flow::error::invalid_access_token%]",
+      "unknown": "[%key:common::config_flow::error::unknown%]"
+    },
+    "step": {
+      "user": {
+        "data": {
+          "access_token": "[%key:common::config_flow::data::access_token%]"
+        },
+        "description": "Get your access token from the barry app under the API section! Check the instructions on https://developer.barry.energy",
+        "title": "Insert your API Token"
+      },
+      "metering_point": {
+        "data": {
+          "metering_point": "Metering Point"
+        },
+        "description": "Choose the address you would like to get spot prices for.",
+        "title": "Almost There!"
```

Comment:
Lets remove this title, and keep it clean.

```suggestion
```

Reply:
Actually this title (Almost There!) is written intentionally so that customer gets to know that there are not many steps and they almost completed the setup process.

Reply:
I understand, but it is not consistent with how it is done in our interface, thus please remove.